frontend/src/components/common/LogRecordsTable.tsx

```diff
@@ -0,0 +1,71 @@
+import React from "react";
+import {
+  Table,
+  TableContainer,
+  Tbody,
+  Td,
+  Th,
+  Thead,
+  Tr,
+} from "@chakra-ui/react";
+import { LogRecord } from "../../types/LogRecordTypes";
+
+type Props = {
+  logRecords: LogRecord[];
+};
+
+const LogRecordsTable = ({ logRecords }: Props): React.ReactElement => {
+  return (
+    <div>
+      <TableContainer marginTop="12px" height="70vh" overflowY="scroll">
+        <Table
+          variant="simple"
+          style={{ minHeight: "400px", verticalAlign: "middle" }}
+        >
+          <Thead className="table-header">
+            <Tr>
+              <Th>Date</Th>
+              <Th>Time</Th>
+              <Th>Resident</Th>
+              <Th>Note</Th>
+              <Th>Employee</Th>
+              <Th>Attn To</Th>
+            </Tr>
+          </Thead>
+
+          <Tbody>
+            {/* TODO: replace mockRecords with logRecords */}
+            {logRecords.map((record) => {
+              // TODO: Investigate alternative methods for date storage + creation
+              const dateObj = new Date(record.datetime);
+
+              return (
+                <Tr key={record.logId} style={{ verticalAlign: "middle" }}>
+                  <Td width="5%">
+                    {`${dateObj.getMonth()} ${dateObj.getDate()} ${dateObj.getFullYear()}`}
+                  </Td>
+                  <Td width="5%">
+                    {`${dateObj.getHours()}:${dateObj.getMinutes()}`}
+                  </Td>
+                  {
+                    // TODO: Resolve the resident record at some point
+                  }
+                  <Td width="5%">{record.residentFirstName}</Td>
+                  <Td whiteSpace="normal" width="75%">
+                    {record.note}
+                  </Td>
+                  <Td width="5%">{record.employeeFirstName}</Td>
```

Comment:
super weird that in the designs we only use first names here, but we should be displaying both since there might be a case where two employees have the first name.

Reply:
Resolved